src/fmt.rs

```diff
@@ -251,12 +290,9 @@ mod test {
         let mut toml = input.parse::<Document>().unwrap();
         fmt_toml(&mut toml, &Config::new());
         #[cfg(target_os = "windows")]
-        assert_eq!(
-            input.replace("\r\n", "\n"),
-            toml.to_string_in_original_order().replace("\r\n", "\n")
-        );
+        assert_eq!(input.replace("\r\n", "\n"), toml.to_string().replace("\r\n", "\n"));
         #[cfg(not(target_os = "windows"))]
-        assert_eq!(input, toml.to_string_in_original_order());
+        assert_eq!(input, toml.to_string());
     }
 
     #[cfg(target_os = "windows")]
```

Comment:
Sorry github won't let me click on line 305 should be
```
"[package]\nname = \"priv-test\"\nversion = \"0.1.0\"\nedition = \"2021\"\nresolver = \"2\"\n# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html\n\n[dependencies]\nstructopt = \"0.3\"\n"
```

(Remove the `\r` as you suggested)